deepdrivemd/api.py

```diff
@@ -227,15 +232,24 @@ def start_simulations(self) -> None:
 
     @result_processor(topic="simulation")
     def process_simulation_result(self, result: Result) -> None:
+        """Receive a training result and then submit new tasks
+
+        Will always submit a new simulation tasks and an inference or training
+        if the :meth:`handle_simulation_output` sets the appropriate flags."""
         # Log simulation job results
         self.log_result(result, "simulation")
         if not result.success:
+            # TODO (wardlt): Should we submit a new simulation if one fails?
+            # (braceal): Yes, I think so.
```

Comment:
Should we change it so we submit a new task regardless of whether the old one failed? You could just move this logic to after submitting the next task and before we handle the outputs.